src/main/java/org/apache/accumulo/testing/performance/tests/ConditionalMutationsPT.java

```diff
@@ -150,7 +150,9 @@ public static double conditionalMutationsTime(ConditionalWriter cw, long seq) th
 
     long t2 = System.nanoTime();
 
-    return 10000.0 / TimeUnit.NANOSECONDS.toSeconds(t2 - t1);
+    double nanosPerSec = (double) TimeUnit.SECONDS.toNanos(1);
+    // return number of conditions per second
+    return 10000.0 / ((t2 - t1) / nanosPerSec);
```

Comment:
This would be more readable in a new method that can be reused more readably:

```java
return rate(numEvents, startNanos, stopNanos);
```

The method could look something like:

```java
private static double rate(long numEvents, long startNanos, long stopNanos) {
  double nanosPerSec = 1.0E9; // or (double) TimeUnit.SECONDS.toNanos(1)
  double seconds = (t2 - t1) / nanosPerSec;
  return numEvents / seconds;
}
```

Reply:
Alternatively, why not perform the calculation in nanos and convert the rate at the end?

Reply:
> Alternatively, why not perform the calculation in nanos and convert the rate at the end?

Because then you'd risk losing precision scaling down when you divide by the nanosecond duration only to multiply by 1E9 to scale back up to seconds. And also, it's much more readable to simply compute the number of seconds, and literally return `numEvents / seconds` to get the number of events per second, rather than convert `events/ns` to `events/s` by multiplying by `ns/s`.

For reference, the code that works in `events/ns` first:

```java
private static double rate(long numEvents, long startNanos, long stopNanos) {
  double eventsPerNano = numEvents / ((double) t2 - t1);
  double nanosPerSec = 1.0E9; // or (double) TimeUnit.SECONDS.toNanos(1);
  return eventsPerNano * nanosPerSec;
}
```

Every line that differs here from the previous suggestion is more confusing. My earlier suggestion was substantially more straight-forward.

Reply:
What about a method to just convert from nanos to seconds? That might make things even more readable.

Reply:
Addressed in dc41ea0 and 27f49d6